src/renderer/mod.rs

```diff
@@ -173,7 +173,8 @@ impl CmdRenderer {
                 return Ok(());
             } else {
                 error!(
-                    "The command `{}` wasn't found, is the `{}` backend installed?",
+                    "The command `{}` wasn't found, is the `{}` backend installed? \
+                    It isn't marked as optional.",
```

Comment:
May I suggest something like:

```
If you want to ignore this error when the `{}` backend is not installed, 
set `optional = true` in the `[output.{}]` section of the book.toml configuration file.
```

Reply:
I felt it would be too verbose to have a longer message. If you prefer it, though, sure.

Reply:
Yea, I generally prefer the longer error message. It can help guide the user to a fix, and provide some more discovery and context.

Reply:
What do you think of this new version?